mod-source-record-manager-server/src/main/java/org/folio/services/flowcontrol/RawRecordsFlowControlServiceImpl.java

```diff
@@ -49,31 +53,31 @@ public class RawRecordsFlowControlServiceImpl implements RawRecordsFlowControlSe
 
   @PostConstruct
   public void init() {
-    LOGGER.info("init:: Flow control feature is {}", enableFlowControl ? "enabled" : "disabled");
+    LOGGER.info("init:: Flow control feature is {}, instanceId: {}", enableFlowControl ? "enabled" : "disabled", instanceId);
   }
 
   /**
-   * This method schedules resetting state of flow control. By default, it triggered each 2 mins.
+   * This method schedules resetting state of flow control. By default, it triggered each 1 mins.
    * This is distributed system and count of raw records pushed to process can be not corresponding
    * with DI_COMPLETE/DI_ERROR, for example because of imports are stuck.
    *
    * For example, batch size - 50 records, max simultaneous - 1 batch, records threshold - 25.
    * Flow control receives 50 records (1 batch) and pause consumers, 20 DI_COMPLETE events came, 30 was missed due to some DB issue.
-   * Current state would equals to 30 and threshold will never met 24 records so consumers will be paused forever
+   * Current state would equals to 30 and threshold will never met 25 records so consumers will be paused forever
    * and as a result all subsequent imports will not have a chance to execute.
    *
    * This scheduled method to reset state intended to prevent that. The correlation between max simultaneous and records
    * threshold can be missed during resetting that can cause that resume/pause cycle may not be as usual, because we are
    * starting from clear state after reset, but any events would not be missed and consumers never pause forever.
    */
-  @Scheduled(initialDelayString = "PT1M", fixedRateString = "${di.flow.control.reset.state.interval:PT2M}")
+  @Scheduled(initialDelayString = "PT1M", fixedRateString = "${di.flow.control.reset.state.interval:PT1M}")
   public void resetState() {
     if (!enableFlowControl) {
       return;
     }
 
     currentState.forEach((tenantId, counterVal) -> {
-      LOGGER.info("resetState:: Tenant: [{}] ", tenantId);
+      LOGGER.info("resetState:: Tenant: [{}], InstanceIs:{} ", tenantId, instanceId);
```

Comment:
should the log be `InstanceId:` instead of `InstanceIs:`

other locations have this potential issue as well.

Reply:
oh, thanks. I added a unique id because now in logs I can't separate log messages from different SRM instances